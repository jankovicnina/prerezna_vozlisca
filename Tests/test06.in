36 40
6 30 0.46
1 3 -1.71
12 10 1.38
9 12 6.7
35 2 7.55
14 24 5.75
32 5 -1.02
16 0 -1.54
33 1 8.69
29 18 4.9
17 32 8.57
8 22 -0.13
15 28 1.17
18 17 -0.13
21 23 5.65
34 25 7.77
24 30 7.29
0 8 2.91
24 20 -0.16
20 7 1.99
5 15 -0.41
2 11 1.83
11 14 -0.15
31 27 1.34
25 33 9.06
10 4 3.14
21 34 9.96
13 16 8.78
27 5 5.44
3 16 4.17
32 6 4.01
19 29 0.85
7 21 0.57
26 23 -0.62
4 26 -0.1
30 35 0.99
22 13 2.74
13 31 3.77
23 19 9.65
28 9 5.91
