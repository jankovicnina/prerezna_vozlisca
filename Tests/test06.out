92 294 422 457 497 617 1001 1144 1185 1489 1824 1932 1969 1984 2177 2363
